boot/freeldr/freeldr/arch/i386/pcvideo.c

```diff
@@ -1111,15 +1111,20 @@ PcVideoSync(VOID)
 VOID
 PcVideoPrepareForReactOS(IN BOOLEAN Setup)
 {
+	PcVideoSetMode80x50_80x43();
```

Comment:
Indentation problem here, should be 4 spaces instead of tabs.

Reply:
Damn!